src/jukebox/components/rfid/hardware/generic_nfcpy/setup.inc.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+CURRENT_USER="${SUDO_USER:-$(whoami)}"
+
+modprobe_file="/etc/modprobe.d/disable_driver_jukebox_nfcpy.conf"
+
+if [ -e "$modprobe_file" ]; then
+    sudo rm -f "$modprobe_file"
+fi
+if lsmod | grep "pn533_usb"; then
+    sudo rmmod pn533_usb 2>/dev/null
+    sudo sh -c "echo 'install pn533_usb /bin/true' >> $modprobe_file"
+fi
+
+if lsmod | grep "port100"; then
+    sudo rmmod port100 2>/dev/null
+    sudo sh -c "echo 'install v /bin/true' >> $modprobe_file"
```

Comment:
Is this correct with only a `v`?

Reply:
This looks like a mis-pressed "ctrl-v". Not really sure how it got there... Good catch